src/Storages/StorageMergeTree.cpp

```diff
@@ -1002,6 +1121,97 @@ void StorageMergeTree::loadMutations()
         increment.value = std::max(increment.value.load(), current_mutations_by_version.rbegin()->first);
 }
 
+void StorageMergeTree::readExportPartitionManifests()
+{
+    for (const auto & disk : getDisks())
+    {
+        for (auto it = disk->iterateDirectory(relative_data_path); it->isValid(); it->next())
```

Comment:
we need a cleanup of old ones

Reply:
Initially, I was deleting the manifests as soon as the commit file was uploaded. But then we changed the requirements so that a partition could be exported only once. To be able to lock these partitions upon re-start, I opted for leaving the export manifests.

If we change that requirement, then I'll delete it for sure.